lightning/src/ln/channelmanager.rs

```diff
@@ -1110,6 +1110,13 @@ impl<ChanSigner: ChannelKeys, M: Deref, T: Deref, K: Deref, F: Deref> ChannelMan
 					if msg.cltv_expiry > cur_height + CLTV_FAR_FAR_AWAY as u32 { // expiry_too_far
 						break Some(("CLTV expiry is too far in the future", 21, None));
 					}
+					// We should be safe against unintentional channel-closure by our onchain logic due to this later
+					// offering a LATENCY_GRACE_PERIOD_BLOCKS. But to make our policy coherent with reception, apply
+					// same delay.
```

Comment:
If I understand this comment, rephrase to:

"In theory, we would be safe against unintentional channel-closure if we only required a margin of LATENCY_GRACE_PERIOD_BLOCKS. But, to be consistent with our above policy for receiving payments, we apply the same delay in both cases."